src/ToricVarieties/cohomCalg/auxilliary.jl

```diff
@@ -62,7 +62,7 @@ function contributing_denominators(variety::AbstractNormalToricVariety)
     
     # ambiguous monomial contributions found during execution?
     if last(output_string_reduced) != "There are no ambiguous contribution monomials to the variety."
-        throw(ArgumentError("cohomCalg experienced ambiguous monomial contributions."))
+        throw(ArgumentError("cohomCalg experienced ambiguous monomial contributions"))
```

Comment:
Maybe something like
```suggestion
        throw(ArgumentError("cohomCalg encountered ambiguous monomial contributions"))
```
? "experienced" to me sounds like cohomCalg developing feelings. ;)

Reply:
If my recollection is correct, then I copied the error message that `cohomCalg` would print out. In any case, I agree that "encountered" is better than "experienced".